packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/disable-security-image/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Disable the security image and additional self-service recovery options
+---
+
+With Okta, new users are prompted to choose a security image and optionally an additional security question prompt when they first sign in. To provide your users with a seamless and smooth sign-in experience, you may need to disable these options.
+
+1. In the Admin Console, go to **Settings** and then **Customization**.
```

Comment:
developer console vs classic ui

Reply:
@briandemers-okta - They can't use the Developer Console for these steps. They have to use the Admin Console (Classic UI). I will put a note at the very top about performing these steps in the Admin Console so that users aren't confused.